homeassistant/components/livisi/coordinator.py

```diff
@@ -58,6 +58,10 @@ async def _async_update_data(self) -> list[dict[str, Any]]:
         except ClientConnectorError as exc:
             raise UpdateFailed("Failed to get LIVISI the devices") from exc
 
+    def _async_dispatcher_send(self, data, event, source):
+        if data is not None:
+            async_dispatcher_send(self.hass, f"{event}_{source}", data)
```

Comment:
Can we add type hints?
And (nit-pick) maybe it makes more sense to change the parameter order (event, source, data)

Reply:
okay.